im21door.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-bitwise */
+// This file relies heavily on the use of bitwise-AND (&) and bitwise-OR (|)
+
+// Bits from section 5 of https://drive.google.com/drive/folders/1XtnRY1ri_p0hvTmEYqqxfRy2tkf6x0jL
+const IM21_SIGNAL_BIT = {
+  HEARTBEAT: 8, // binary 1000
+  BATTERY: 4, // binary 0100
+  STATE: 2, // binary 0010
+  TAMPERED: 1, // binary 0001
+}
+
+// Expects to be given a hexidecimal string
+// Returns a boolean
+function isLowBattery(signal) {
+  const intSignal = parseInt(signal, 16)
+  return (intSignal & IM21_SIGNAL_BIT.BATTERY) > 0
+}
+
+// Expects to be given a hexidecimal string
+// Return s a boolean
+function isOpen(signal) {
+  const intSignal = parseInt(signal, 16)
+  return (intSignal & IM21_SIGNAL_BIT.STATE) > 0
+}
+
```

Comment:
Thoughts on checking the tamper bit and making a sentry log message when its 1?

Reply:
Sure. I'd be up for that. I don't know what exactly it would mean to be tampered with, but just adding a Sentry log will let us maybe figure it out if it ever happens.

Reply:
Done.